Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -405,9 +411,31 @@ void vtkMRMLApplicationLogic::PropagateTableSelection()
       continue;
       }
     tnode->SetTableNodeID( tableId );
-    }
+  }
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLApplicationLogic::PropagatePlotLayoutSelection()
```

Comment:
here I mean the vtkMRMLPlotLayoutNode. I think just calling Plot can be misleading to a vtkMRMLPlotNode, no?